src/Test/QuickCheck/Classes.hs

```diff
@@ -717,23 +718,28 @@ arrowChoice = const ("arrow choice laws"
     rightMovesP f g = (left f >>> right (arr g))
                         =-= ((right (arr g)) >>> left f)
 
-traversable :: forall f a b m.
-               ( Traversable f, Monoid m, Show (f a)
-               , Arbitrary (f a), Arbitrary b, Arbitrary m
-               , CoArbitrary a
-               , EqProp (f b), EqProp m) =>
-               f (a, b, m) -> TestBatch
+traversable :: forall t a b c f.
+               ( Traversable t, Applicative f
+               , Arbitrary (t a), Arbitrary (t b), Arbitrary (f b), Arbitrary (f c)
+               , CoArbitrary a, CoArbitrary b
+               , Show (t a), Show (t b)
+               , EqProp (t b), EqProp (f (f (t c)))) =>
+               t (f a, b, c) -> TestBatch
 traversable = const ( "traversable"
-                    , [ ("fmap", property fmapP)
-                      , ("foldMap", property foldMapP)
```

Comment:
What's the reason for removing these laws? Are they implied by identity and composition?

I noticed that both `quickcheck-classes` and `hedgehog-classes` test these laws too, so I'm inclined to keep them.

Reply:
Sorry, that was a mistake! I'll bring them back.